webview/src/experiments/components/table/body/TableBody.tsx

```diff
@@ -16,7 +16,6 @@ interface TableBodyProps extends RowProp, InstanceProp, BatchSelectionProp {
 export const TableBody: React.FC<TableBodyProps> = ({
   row,
   instance,
-  contextMenuDisabled,
```

Comment:
[F] This starts here and drills all the way down but is not set on `<Table>`. I assumed that means I can remove it.